go.mod

```diff
@@ -1,20 +1,11 @@
 module github.com/oamg/rhc-worker-bash
 
-go 1.20
+go 1.16
 
 require (
 	git.sr.ht/~spc/go-log v0.0.0-20230531172318-1397be06f5f4
 	github.com/google/uuid v1.3.0
 	github.com/redhatinsights/yggdrasil v0.0.0-20210811162724-41397343c25b
-	google.golang.org/grpc v1.56.2
+	google.golang.org/grpc v1.51.0
```

Comment:
The version here needs to be `v1.51.0` as this is the latest one that works for us under RHEL 7. After that, `v1.52.0+` the grpc dependency will depend on a newer version of `golang.org/x/sys`, which causes the code to break on go1.16 (the latest version for RHEL7)

https://github.com/grpc/grpc-go/blob/v1.51.0/go.mod#L16

Reply:
ok, makes sense